src/main/java/Yukee/Yukee.java

```diff
@@ -0,0 +1,246 @@
+package Yukee;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+import Yukee.task.Task;
+import Yukee.task.Todo;
+import Yukee.task.Deadline;
+import Yukee.task.Event;
+import Yukee.exception.YukeeException;
+import Yukee.exception.EmptyDescriptionException;
+import Yukee.exception.InvalidCommandException;
+
+public class Yukee {
+    private static final String FILE_PATH = "./src/main/java/data/Yukee.txt";
+    private static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        String logo = """
+                 Y   Y  U   U  K   K  EEEEE  EEEEE
+                  Y Y   U   U  K  K   E      E
+                   Y    U   U  KKK    EEEE   EEEE
+                   Y    U   U  K  K   E      E
+                   Y     UUU   K   K  EEEEE  EEEEE
+                """;
+        System.out.println("Hello! I'm Yukee, your friendly assistant!\n" + logo);
+
+        loadTasksFromFile();
+
+        System.out.println("What can I do for you today?");
+
+        while (true) {
+            try {
+                System.out.print("Enter command: ");
+                String input = scanner.nextLine().trim();
+                String[] inputSplit = input.split(" ", 2);
+                String command = inputSplit[0].toLowerCase();
+
+                switch (command) {
+                    case "bye":
+                        System.out.println("Bye! Hope to see you again soon!");
+                        return;
+
+                    case "list":
+                        if (tasks.isEmpty()) {
+                            System.out.println("Your task list is empty! Try adding some tasks!");
+                        } else {
+                            System.out.println("Here are the tasks in your list:");
+                            for (int i = 0; i < tasks.size(); i++) {
+                                System.out.println((i + 1) + ". " + tasks.get(i));
+                            }
+                        }
+                        break;
```

Comment:
The code here seems quite nested. Consider splitting it into different functions.